dotcom-rendering/src/web/components/CommentCount.importable.tsx

```diff
@@ -1,14 +1,17 @@
 import { css } from '@emotion/react';
 
+import { joinUrl } from '@guardian/libs';
 import { textSans, between, until } from '@guardian/source-foundations';
 
 import { formatCount } from '@root/src/web/lib/formatCount';
 import CommentIcon from '@frontend/static/icons/comment.svg';
+import { decidePalette } from '../lib/decidePalette';
+import { useDiscussion } from '../lib/useDiscussion';
 
 type Props = {
-	palette: Palette;
-	isCommentable: boolean;
-	commentCount?: number;
+	format: ArticleFormat;
+	discussionApiUrl: string;
+	shortUrlId: string;
```

Comment:
Previously we passed in the values to display, now we pass in the things the component needs to make the api call to fetch the values itself